apollo-integration/src/test/java/com/apollographql/apollo/RxApolloTest.java

```diff
@@ -88,7 +92,7 @@
     TestSubscriber<EpisodeHeroName.Data> subscriber = new TestSubscriber<>();
     Subscription subscription = RxApollo
         .from(call)
-        .delay(5, TimeUnit.SECONDS)
+        .delay(RX_DELAY_SECONDS, TimeUnit.SECONDS)
```

Comment:
The docs for delay say:

> This version of delay operates by default on the computation Scheduler.

which means that its a non blocking call and hence wont block the thread running the test. So the reason behind introducing delay is to let the observable do work in another thread while in the main thread we try to unsubscribe from the it. Basically simulate a real world scenario of unsubscribing from a Subscription.